src/ui/primitives/drawer.tsx

```diff
@@ -50,8 +50,11 @@ function DrawerOverlay({
 function DrawerContent({
   className,
   children,
+  hideDragHandle,
   ...props
-}: React.ComponentProps<typeof DrawerPrimitive.Content>) {
+}: React.ComponentProps<typeof DrawerPrimitive.Content> & {
+  hideDragHandle?: boolean
```

Comment:
maybe it should be displayDragHandle={false} (you want to avoid double negative)